Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php

```diff
@@ -148,10 +140,12 @@ private function injectAlignmentPlaceholders(Tokens $tokens, $startAt = null, $e
             }
 
             if ($token->equals(',')) {
-                do {
+                for ($i = $index; $i < $endAt - 1; ++$i) {
```

Comment:
@SpacePossum This `for` statement is the reason of https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1450 issue.

Revert this part by the old while solve it. Any idea?